trendshop/templates/registration/password_reset_email.html

```diff
@@ -1,8 +1,10 @@
 {% autoescape off %}
-To initiate the password reset process for your {{ user.get_username }} Foodmania Account,
+Hello {{ user.first_name }},
+
+To initiate the password reset process for your {{ user.get_username }} You & Me Shop Account,
 click the link below:
 
-{{ protocol }}://{{ domain }}{% url 'user:password_reset_confirm' uidb64=uid token=token %}
+{{ protocol }}://192.168.0.190:8000{% url 'user:password_reset_confirm' uidb64=uid token=token %}
```

Comment:
why take static domain ?

Reply:
i was checking from my computer's IP because in variable domain it was coming with example.com and link was getting invalid.